### PR TITLE
Prep for catapult release.

### PR DESCRIPTION
**Pull request by mikelehen**
@jwngr PTAL?

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+fixed - Disable progress bar when piping the output.
```
Comment (jwngr) [inline changelog.txt]: Small nit: use past tense. `s/Disable/Disabled`

**Comment by jwngr** [comment]
One small comment. Back to you.

**Comment by mikelehen** [comment]
Change made and merged.  Thanks @jwngr!

**Comment by jwngr** [comment]
:boar: looks good

**Pull request merged**